tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1152,6 +1153,48 @@ def test_external_new_user_login_and_check_count_rhsso(
         assert error.typename == "NavigationTriesExceeded"
 
 
+@pytest.mark.skip_if_open("BZ:1873439")
+@destructive
+def test_login_failure_rhsso_user_if_internal_user_exist(
+    enable_external_auth_rhsso, rhsso_setting_setup, session, module_org, module_loc
+):
+    """Verify the failure of login for the external rhsso user in case same username
+    internal user exists
+
+    :id: e573902c-ed1a-11ea-835a-d46d6dd3b5b2
+
+    :BZ: 1873439
+
+    :CaseImportance: High
+
+    :steps:
+        1. create an internal user
+        2. create a rhsso user with same username mentioned in internal user
+        3. update the satellite to use rhsso and now try login using external rhsso user
+
+    :expectedresults: external rhsso user should not able to login with same username as internal
```

Comment:
Are we sure this is what will happen in this scenario?

I only skimmed over bz and upstream fix, but I'm under impression that the way it will be fixed and exact details on how it will work are still being discussed.

Of course we can still merge and if fix turns out to be different, then raise another PR with fix.

Reply:
Yeah, you are correct based on the upstream PR, We are ahead of with our testing. We can raise fix when we see this test fails and tracked too.

Reply:
As per my understanding the User should not able to login with message "User is already exist with same Username" in other LDAP Auth Sources.